docs/concepts/Security-Best-Practices.md

```diff
@@ -0,0 +1,166 @@
+
+For more information on managing NuGet dependencies, [see the following documentation](https://docs.microsoft.com/nuget/consume-packages/overview-and-workflow).
+
+### GitHub Dependency Graph 
+
+You can use GitHub’s dependency graph to see the packages your project depends on and the repositories that depend on it. This can help you see any vulnerabilities detected in its dependencies.
+
+For more information on GitHub repository dependencies, [see the following documentation](https://github.co/dependency-graph).
+
+### Dependency Versions
+
+To ensure a secure supply chain of dependencies, you will want to ensure that all your dependencies are regularly updated to the latest stable version. Your dependencies can include code you depend on, binaries you consume, tooling you use, and other components. This can include:
+
+-	[Visual Studio](https://visualstudio.microsoft.com/downloads/)
+-	[.NET SDK & Runtime](https://dotnet.microsoft.com/download)
+-	[NuGet](https://www.nuget.org/downloads)
+-	[NuGet Packages](https://docs.microsoft.com/nuget/consume-packages/reinstalling-and-updating-packages)
+
+## Manage your dependencies
+
+### NuGet Deprecated & Vulnerable Dependencies
+
+You can use the [dotnet CLI](https://docs.microsoft.com/dotnet/core/tools/dotnet-list-package) to list any known deprecated or vulnerable dependencies you may have inside your project or solution. You can use the command `dotnet list package --deprecated` or `dotnet list package --vulnerable` to provide you a list of any known deprecations or vulnerabilities.
+
+### GitHub Vulnerable Dependencies
+
+If your project is hosted on GitHub, you can leverage [GitHub Security](https://docs.github.com/en/free-pro-team@latest/github/finding-security-vulnerabilities-and-errors-in-your-code/automatically-scanning-your-code-for-vulnerabilities-and-errors) to find security vulnerabilities and errors in your project and Dependabot will fix them by opening up a pull request against your codebase. 
+
+Catching vulnerable dependencies before they are introduced is one goal of the [“Shift Left”](https://en.wikipedia.org/wiki/Shift-left_testing) movement. Being able to have information about your dependencies such as their license, transitive dependencies, and the age of dependencies helps you do just that.
+
+For more information about Dependabot alerts & security updates, [see the following documentation](https://docs.github.com/en/github/managing-security-vulnerabilities/about-alerts-for-vulnerable-dependencies).
+
+### NuGet Feeds
+
+Packages can come from different feeds. To ensure you are secure, knowing what feed your packages are coming from is a best practice. One such best practice is the use of a single feed. You can accomplish this by using multiple upstream source feeds to bring your packages into a single feed.
```

Comment:
More explanation is needed here on why multiple feeds is a problem. Also, using AzureDevOps is not something most our customers do, so this can't be the only recommendation we provide. 
Lock files and Client Trust policies help mitigate this (even if not 100%). We should frame it accordingly.

Reply:
@JonDouglas , ping on this comment.

"When using multiple NuGet feeds, a package can be downloaded from any of the feeds. To ensure your build is predictable, we recommend using a single feed or a private feed with upstreaming capabilities. For further information see [link to whitepaper]"

Reply:
Also, a recommendation from the whitepaper that doesn't appear here is to ensure packages on private feeds have a namespace, and that this namespace is reserved on nuget.org

Reply:
I'll reword this section now that we have a whitepaper out. Thanks for the suggestion!

Reply:
Added a commit